mqtt.proxy.asyncapi/README.md

```diff
@@ -79,11 +80,13 @@ brew install mosquitto
 
 ### Verify behavior
 
-Connect a subscribing client to mosquitto broker to port `1884`. Using mosquitto_pub client publish `{"id":"1","status":"on"}` to Zilla on port `1883`. Verify that the message arrived to on the first client.
+Connect a subscribing client to mosquitto broker to port `1883`. Using mosquitto_pub client publish `{"id":"1","status":"on"}` to Zilla on port `7183`. Verify that the message arrived to on the first client.
 ```bash
-mosquitto_sub -V '5' -t 'smartylighting/streetlights/1/0/event/+/lighting/measured' -d -p 1884
+mosquitto_sub -V '5' -t 'smartylighting/streetlights/1/0/event/+/lighting/measured' -d -p 1883
```

Comment:
Note that `mosquitto_sub` is designed not to require the port if standard port 1883 is used, so `-p 1883` can be omitted here.